test/docker_e2e.sh

```diff
@@ -62,8 +62,8 @@ docker run -d --rm --name dss-crdb-for-debugging \
 
 sleep 5
 echo " -------------- BOOTSTRAP ----------------- "
-echo "Building local container for testing"
-docker build -q --rm . -t local-interuss-dss-image
```

Comment:
This does print a lot of stuff, I find it too verbose

Reply:
Yeah, I'm piping the output into a log file rather than silencing it entirely so that build issues can be quickly debugged; I agree we don't want to dump it to stdout here.

Reply:
I'm assuming that piping the `docker build` output to `grpc-backend-build.log` is sufficient to address the verbosity concern, but will open another PR to adjust if not.